Docs/Debugging/Debugging with Eclipse and pyOCD.md

```diff
@@ -0,0 +1,68 @@
+#Debugging with Eclipse and pyOCD
+
+This article explains how to debug an exported mbed project with Eclipse and pyOCD. To debug a project, you need to first export it from mbed as a makefile project, import the makefile project into Eclipse and finally setup debugger configurations. Below are these steps but in much more detail.
+
+##Initial Setup
+1. Download the latest version of eclipse with CDT.
+1. Install the GNU ARM Eclipse plugin.
+	1. Open Eclipse.
+	1. Click the **Help** menu item and select **Install New Software**.
+	1. In the **Work with** box paste the install address and press enter - [http://redmine.laoslaser.org/boards/3/topics/635](http://sourceforge.net/projects/gnuarmeclipse/files/Eclipse/updates/).
+  1. The package **GNU ARM C/C++ Cross Development Tools** should appear. Select it.
+  1. Click **Next** repeatedly and accept licence agreements.
+  1. Click **Finish**. If prompted to restart Eclipse, select **Yes**.
+1. [Install tools necessary for makefile projects](https://developer.mbed.org/handbook/Exporting-to-Make):
+  1. [GCC ARM Embedded](https://launchpad.net/gcc-arm-embedded).
+  1. Windows only
+    1. Add GCC ARM Embedded bin directory to path after installing - Ex. C:\Program Files (x86)\GNU Tools ARM Embedded\4.9 2015q2\bin.
+    1. Install **make** and add it to path. Make is available as one of the tools here: [http://gnuwin32.sourceforge.net/](http://gnuwin32.sourceforge.net/).
```

Comment:
The link is correct but isn't relevant to mbed 5.0. @c1728p9 says the 5.0 document is almost identical to this one and can be found here: https://developer.mbed.org/users/c1728p9/notebook/debugging-mbed-50-projects-with-eclipse-and-pyocd/.